test/unit/org/apache/cassandra/db/repair/CompactionManagerGetSSTablesForValidationTest.java

```diff
@@ -105,7 +106,7 @@ private void registerRepair(boolean incremental) throws Exception
     {
         sessionID = nextTimeUUID();
         Range<Token> range = new Range<>(MT, MT);
-        ActiveRepairService.instance.registerParentRepairSession(sessionID,
+        ActiveRepairService.instance().registerParentRepairSession(sessionID,
                                                                  coordinator,
```

Comment:
super-nit: The addition of `()` here and in a bunch of other places bumps the horizontal alignment of the argument list.

Reply:
used IntelliJ to ask for all callers and fixed... leaving this open just as a reminder